src/tools/dashboard/src/util.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! The types and procedures in this modules are similar to the ones in
+//! `compiletest`. Consider using `Litani` to run the test suites.
+
```

Comment:
We should import these types and procedures instead. Or add them here and import in `compiletest`.

Reply:
Opened an issue for this.